src/cow-react/common/hooks/useParameterizedMainMenu.ts

```diff
@@ -0,0 +1,20 @@
+import { BasicMenuLink, isBasicMenuLink, MAIN_MENU, MenuTreeItem } from 'constants/mainMenu'
+import { useParameterizeLimitOrdersRoute } from 'cow-react/modules/limitOrders/hooks/useParameterizeLimitOrdersRoute'
+import { Routes } from 'constants/routes'
+import { useMemo } from 'react'
+
+export function useParameterizedMainMenu(): MenuTreeItem[] {
+  const limitOrdersRouteWithParams = useParameterizeLimitOrdersRoute()
+
+  return useMemo(() => {
+    const copy: MenuTreeItem[] = JSON.parse(JSON.stringify(MAIN_MENU))
```

Comment:
why not using it directly `MAIN_MENU`?

Reply:
Fixed here: https://github.com/cowprotocol/cowswap/pull/1134